policyengine_us/variables/gov/states/id/tax/income/deductions/id_salt_deduction.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class id_salt_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho SALT deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=8"
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        salt = tax_unit("state_and_local_sales_or_income_tax", period)
```

Comment:
use a different name - salt includes property taxes. also accordingly change the parameter,

would this simplification work? not immediately understanding the purpose of the written logic

```suggestion
        salt = add(tax_unit, period, ["state_and_local_sales_or_income_tax", "real_estate_taxes"])
        p = parameters(
            period
        ).gov.irs.deductions.itemized.salt
        cap = p.cap[tax_unit("filing_status", period)]
        return min_(salt, cap)
```

Reply:
It would not as we need the amount from line 5A (just state and local income taxes) if (state and local income tax + real estate tax) < federal cap
And if > cap:
real estate tax - cap

[Instructions](https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=8)
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/8dbe57b7-9cab-4406-b2c5-295a2b6cda9d)